docs/tutorials/torch_qgan_test.py

```diff
@@ -0,0 +1,202 @@
+# Necessary imports
```

Comment:
What is the purpose of this file ? Until now we only have support python files under docs/tutorials if we need to define methods that will be imported by tutorials themselves but it seems this is a file that would be run by users ? It won't show in any documentation, contrary to notebooks, so users won't know about it.
In any case, it is missing the IBM copyright. Also, could you please add the block at the end:
```
if __name__ == "__main__":
    _main()
 ```
and put all the global executable code inside the _main() function ? This is to guard it in case this file is loaded more than once by multlthread/multiprocessing code in QIskit Terra, besides being good python practice.

Reply:
This file shouldn't be in there. Thanks for pointing this out.